docs/guides/es2k/running-infrap4d.md

```diff
@@ -0,0 +1,281 @@
+# Run Infrap4d on Intel IPU E2100
+
+- [1. Overview](#1-overview)
+- [2. Running Infrap4d on Arm Compute Complex (ACC)](#2-running-infrap4d-on-arm-compute-complex-acc)
+   - [2.1 Extract the tarball to get P4 libraries](#21-extract-the-tarball-to-get-P4-libraries)
+   - [2.2 Setup the Environment](#22-setup-the-environment)
+   - [2.3 Install TLS Certificates](#23-install-tls-certificates)
+   - [2.4 Generate Forwarding Pipeline Binary](#24-generate-forwarding-pipeline-binary)
+      - [2.4.1 Prepare configuration files](#241-prepare-configuration-files)
+      - [2.4.2 Generate binary](#242-generate-binary)
+   - [2.5 Start Infrap4d](#25-start-infrap4d)
+   - [2.6 Set the Pipeline](#26-set-the-pipeline)
+   - [2.7 Configure Forwarding Rule to Receive Traffic](#27-configure-forwarding-rule-to-receive-traffic)
+   - [2.8 Test Traffic from Link Partner to E2100](#28-test-traffic-from-link-partner-to-e2100)
+
+## 1. Overview
+
+This document explains how to install and run the Infrap4d
+on Intel IPU E2100 hardware.
+
+Note: This document assumes that you have followed
+[Deploying P4 programs guide](https://github.com/ipdk-io/networking-recipe/blob/main/docs/guides/es2k/deploying-p4-programs.md)
+to install the P4 package on FXP during IMC boot-up.
+
+## 2. Running Infrap4d on Arm Compute Complex (ACC)
+
+### 2.1 Extract the tarball to get P4 libraries
+
+Login to ACC and untar the `p4.tar.gz` tarball available under `/opt`
+
+```bash
+[root@mev-acc-rl opt]# cd /opt
+[root@mev-acc-rl opt]# tar -xf p4.tar.gz
+[root@mev-acc-rl opt]# cd /opt/p4/
+[root@mev-acc-rl p4]# ls -lr
+total 8
```

Comment:
Is this output truncated? shows total 8 and lists 2

Reply:
No, ls -lr shows total no. of disk blocks consumed and not total no. of files. Please see https://askubuntu.com/questions/810563/why-does-ls-l-count-more-files-than-me for more information